step-definitions/core/vue.js

```diff
@@ -35,20 +32,67 @@ Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart,
   }, [], function(result) {
     this.assert.ok(result.value);
   });
+})
+
+
+Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart, accountName) => {
+  const evan = setupEvan(client);
+
+
+
+  await client.url(`${ evan.baseUrl }#/dashboard.vue.evan`);
+  await client.execute(function() {
+    window.localStorage.setItem('evan-vault', '');
+    window.localStorage.setItem('evan-test-mode', true);
+    window.localStorage.setItem('evan-warnings-disabled', '{"payment-channel":true}');
+    window.localStorage.setItem('evan-language', 'en');
+    window.localStorage.setItem('evan-test-recaptchaId', '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI');
+    return true;
+  })
+  await client.pause(3000);
```

Comment:
in general pause() slows the whole process and should be replaced where possible by watchers on expected elements like `waitForElementPresent`

Reply:
Makes sense. Feel free to add a suggestion (button in comment box or "```suggestion" in comment. ^^

Reply:
in this case the `await client.waitForElementPresent(`a[href*="sign-in"]`, 60 * 1000);` seems to be enough for me, maybe the time window is just too short?